Resources/Prototypes/_StarLight/Entities/Clothing/Eyes/hud.yml

```
@@ -27,7 +27,7 @@
   parent: [ ClothingEyesBase, ShowSecurityIcons, BaseSecurityContraband ]
   id: ClothingEyesHudBrigmedic
   name: brigmedic hud
-  description: Tamashi-upgraded HUDs. Yes. Brigmedic ones.
+  description: Tamashi-upgraded hud. Yes. Brigmedic ones.
```

```suggestion
  description: Brigmedic hud, differs from another ones in that it contains both medical and security information at the same time.
```

Isn't it's better to make general hud, like you do this with headset or like with mostly another huds?

---

Color scheme differences - brigmedic hud clashes with the intended color scheme of corpsman

---

Description changes made.